packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Documents.Example.tsx

```diff
@@ -271,6 +275,9 @@ export class DetailsListDocumentsExample extends React.Component<{}, IDetailsLis
       if (newCol === currColumn) {
         currColumn.isSortedDescending = !currColumn.isSortedDescending;
         currColumn.isSorted = true;
+        this.setState({
+          announced: <Announced message={`${currColumn.name} is sorted ${currColumn.isSortedDescending ? 'descending' : 'ascending'}`} />
```

Comment:
Minor feedback, but it might be better to store data in state, than a whole component. It would be a better separation of concerns; let the render content render Announced conditionally based on the data state. (when i read line 205, I thought `announced` was a string, not a component.)

```
this.setState({
  announcedMessage: `${currColumn.name} is sorted ${currColumn.isSortedDescending ? 'descending' : 'ascending'}`
})
```